.github/workflows/build.yml

```diff
@@ -2,13 +2,13 @@
 name: build
 
 on:
-  push
+  pull_request: ~
+  push:
+    branches: [master]
```

Comment:
This drops the build for non-master branches...

Reply:
It should still work for pull requests - the reason for this restriction is otherwise you'll end up with two pipeline executions for pull requests branched from seatgeek master. I can put it back if you really want it.